lib/ohloh_scm/adapters/bzr/misc.rb

```diff
@@ -43,7 +43,7 @@ def export(dest_dir, token=head_token)
 			run "cd '#{url}' && bzr export --format=dir -r #{to_rev_param(token)} '#{dest_dir}'"
 		end
 
-    def tags
+		def tags
```

Comment:
This looks like the spacing fell apart. Is this passing Rubocop?

Reply:
Rubocop is a mess with the source code
156 files inspected, 11974 offenses detected

I corrected this change only.